dbt/models/qc/schema.yml

```diff
@@ -486,3 +486,155 @@ models:
+              at_least: 0.9
+              group_by_columns: ['year']
+      - name: deed_type
+        tests:
+          - dbt_utils.not_null_proportion:
+              name: qc_vw_mydec_sales_null_values_deed_type_null_proportion
+              at_least: 0.9
+              group_by_columns: ['year']
+      - name: price
+        tests:
+          - dbt_utils.not_null_proportion:
+              name: qc_vw_mydec_sales_null_values_price_null_proportion
+              at_least: 0.9
+              group_by_columns: ['year']
+      - name: seller
+        tests:
+          - dbt_utils.not_null_proportion:
+              name: qc_vw_mydec_sales_null_values_seller_null_proportion
+              at_least: 0.9
+              group_by_columns: ['year']
+
+  - name: qc.vw_iasworld_sale_rowcount_matches_sales_mydec
+    description: '{{ doc("view_vw_iasworld_sale_rowcount_matches_sales_mydec") }}'
+    config:
+      tags:
+        - test_qc_sales
+    tests:
+      - expression_is_true:
+          name: qc_vw_iasworld_sale_rowcount_matches_sales_mydec_check
+          expression: comparison != 'No significant difference'
+
+  - name: qc.vw_sale_mydec_change_in_high_low_value_sales
+    description: '{{ doc("view_vw_sale_mydec_change_in_high_low_value_sales") }}'
+    config:
+      tags:
+        - test_qc_sales
+    tests:
+      - expression_is_true:
+          name: qc_vw_sale_mydec_change_in_high_low_value_sales_low_value_gt_25%
+          expression: |
+            price_less_than_10k_growth_status != 'No significant change'
+      - expression_is_true:
+          name: qc_vw_sale_mydec_change_in_high_low_value_sales_high_value_gt_25%
+          expression: |
+            price_greater_than_1m_growth_status != 'No significant change'
+
+  - name: qc.vw_iasworld_sale_unmatched_joins_sales_mydec
+    description: '{{ doc("view_vw_iasworld_sale_unmatched_joins_sales_mydec") }}'
+    config:
+      tags:
+        - test_qc_sales
+    tests:
+      - expression_is_true:
+          name: qc_vw_iasworld_sale_unmatched_joins_sales_mydec_unmatched_mydec_sales_lte_1000
+          expression: my_dec_unmatched < 1000
+      - expression_is_true:
+          name: qc_vw_iasworld_sale_unmatched_joins_sales_mydec_unmatched_iasworld_sales_lte_1000
+          expression: iasworld_unmatched < 1000
+
+  - name: qc.vw_iasworld_sale_day_of_month
+    description: '{{ doc("view_vw_iasworld_sale_day_of_month") }}'
+    config:
+      tags:
+        - test_qc_sales
+    tests:
+      - expression_is_true:
+          name: qc_vw_iasworld_sale_day_of_month_lte_half_observations
+          expression: sales_by_day > 0.5 * total_sales
+
+  - name: qc.vw_iasworld_sales_high_value_by_class
+    description: '{{ doc("view_vw_iasworld_sales_high_value_by_class") }}'
+    config:
+      tags:
+        - test_qc_sales
+    tests:
+      - expression_is_true:
+          name: qc_vw_iasworld_sales_high_value_by_class_lte_20000000
+          expression: price > 20000000
```

Comment:
**issue:** This test will never fail since the condition is also specified in the view.

Reply:
I don't have Python up and running now, but the intention was for this to always fail based on the observations produced in the view.

Reply:
I believe the way it was defined would have meant it _would_ fail, but it was redunant. Only prices greater than $20,000,000 would have made it into the view, which would have been detected by this test. Regardless, no need to filter the view on prices.

Reply:
Oh oh, you're right. Still, no need to filter the view.